VIPs/vip-7.md

```diff
@@ -0,0 +1,260 @@
+
+This phase is out of scope for this VIP.
+
+# Selection considerations
+
+The process of selecting appropriate storage nodes requires consideration of:
+
+- Geographic location
+- Availability
+- Performance
+- Regulatory jurisdiction
+- Node operator
+- User preference
+
+The protocol must specify a default selection process for new users. It's intended that advanced users can select their own storage nodes at any time (including selecting self-hosted storage nodes).
+
+## Geographic location
+
+The location of the storage node could consist of the following relevant metadata:
+
+- Country
+- Region (ie: Europe, South East Asia)
+- Datacenter
+- Latitude / Longitude
+
+## Recommendation
+
+1. A registry of all datacenters will be created (`Datacenter Registry`) that has a `uniqueId`, `name`, `countryLocation`, `latitude` and `longitude`.
+2. Each storage node will be included in the `Storage Node Registry` and specify the `uniqueId` of the datacenter where the node is located.
+3. A registry of all regions will be created (`Region Registry`) that maps `country` to `region`. This will be helpful for the client protocol to narrow down the most appropriate storage nodes to consider for selection.
+
+## Availability
+
+The availability of a storage node could consider the following relevant metadata:
+
+- % uptime (ie: 99.99% uptime)
+- Establishment date (ie: created 1 day ago is less trustworthy than created 1,000 days ago)
+- Outages (ie: 5 outages in the last day)
+
+## Recommendation
+
+1. Providing dynamic feedback on uptime and outages is a non-trivial process, so `% uptime` and `outages` is out of scope for now.
+2. `establishmentDate` will be included in the `Storage Node Registry`.
+
+## Performance
+
+The performance of a storage node could consider the following relevant metadata:
+
+- Latency for the current user
+- Average latency for all users
+- Upload and download speed
+
+### Recommendations
+
+1. Verida client implementation will provide a method to easily obtain the `latency` for a storage node (measured in milliseconds).
+2. Providing dynamic feedback on uptime and outages is a non-trivial process, so `average latency` and `up / down speed` is out of scope for now.
+
+## Regulatory jurisdiction
+
+The regulatory jurisdiction of a storage node could consider the following relevant metadata:
+
+- Country where the storage node is located
+- Country of legal entity operating the storage node
+
+Some governments have the right to access digital infrastructure if it is located in a particular country. Similarly some governments have the right to access digital infrastructure, regardless of country the data is located, if the infrastructure operator falls under their jurisdiction.
+
+### Recommendations
+
+1. Require storage nodes to specify their `countryResidence`. This information will not be used other than to display to the user.
```

Comment:
I think this name is confusing. I think the intention is to establish what legal jurisdiction the company running the node is under but legal entities don't really have a "residence" concept. 

I think it could be simplified by calling it `legalJurisdiction` (although obviously this is a complicated topic too). 

Should be a 2-character ISO code.

Reply:
> the ATO talks about your "residency" for tax purposes as do many other countries.

I think this is for an individual, not a company.

Reply:
As per advisor advice, physical location of the node is sufficient.